doc/source/generated/units_table.txt

```diff
@@ -0,0 +1,132 @@
+
```

Comment:
can the script that generates this write a comment at the top to say how to regenerate it?  just a `'#made by gen_unit_tables.py'`

Reply:
Done!